pandas/core/frame.py

```diff
@@ -6827,11 +6827,23 @@ def _dispatch_frame_op(self, right, func: Callable, axis: int | None = None):
         -------
         DataFrame
         """
+        import pandas.core.computation.expressions as expressions
```

Comment:
we import expressions at the top of internals.blocks, so should be safe to do at module level here